tests/cupy_tests/cuda_tests/test_memory.py

```diff
@@ -79,7 +79,15 @@ def check(self, device_id):
         if self.specify_device_id:
             kwargs = {'device_id': device_id}
 
-        unowned_mem = memory.UnownedMemory(*args, **kwargs)
+        if cupy.cuda.runtime.is_hip and self.allocator is memory._malloc:
+            # In ROCm, it seems that `hipPointerGetAttributes()`, which is
```

Comment:
Question: is this new in ROCm 5.0?

Reply:
This test also fails in ROCm 4.3 on my local.